packages/opal-server/opal_server/data/data_update_publisher.py

```diff
@@ -72,6 +74,25 @@ def publish_data_updates(self, update: DataUpdate):
         """
         all_topic_combos = set()
 
+        invalidEntries: List[DataSourceEntry] = []
```

Comment:
Hmm... validation is good although I'm not sure that's the right spot.
Clients can get entries also from `external_source_url`.
I think the best way would be to make those validations in `DataSourceEntry`'s own initialization (and catching the potential exceptions in the proper places)